modules/apps/jaggery/mdm/src/mdm/units/login/login.hbs

```diff
@@ -8,7 +8,7 @@
         <span class="wr-validation-summary hidden">
                 <p>Login failed. Check your credentials and try again.</p>
         </span>
-        <form method="POST" class="form-login-box" action="{{loginPath}}">
+        <form id=login method="POST" class="form-login-box" action="{{loginPath}}" >
```

Comment:
Duly Noted and thank you for the recommendation.
Recommended fixes can be found in PR[1]

[1]. #166